lib/utils.js

```diff
@@ -12,21 +16,61 @@ var basicAuth = require('basic-auth');
  * @param   {string}   password Expected password
  * @returns {function} Express 4 middleware requiring the given credentials
  */
-exports.basicAuth = function(username, password) {
-	return function(req, res, next) {
-
-		if (!username || !password) {
-			console.log('Username or password is not set.');
-			return res.send('<h1>Error:</h1><p>Username or password not set. <a href="https://github.com/alphagov/govuk_prototype_kit/blob/master/docs/deploying.md#3-set-a-username-and-password">See guidance for setting these</a>.</p>');
-		}
-		
-		var user = basicAuth(req);
-
-		if (!user || user.name !== username || user.pass !== password) {
-			res.set('WWW-Authenticate', 'Basic realm=Authorization Required');
-			return res.sendStatus(401);
-		}
-
-		next();
-	};
+function basicAuth (username, password) {
```

Comment:
The name of this function currently clashes with the variable being declared on line 5 for the `basic-auth` lib. Would be better to use a separate name for this function.